### PR TITLE
chore(PROCESS0UT-368): incremental Authorization doc

### PR DESCRIPTION
**Pull request by andrea-vasapollo-cko**
## Description
Incremental Authorization feature

## Jira Issue
https://checkout.atlassian.net/browse/PROCESS0UT-368

### CODE REVIEW

File: source/payments/incremental-authorization.md.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Incremental authorization
+---
+
+# Incrementing an existing authorized invoice
+It is possible to increment the authorization amount of a payment by submitting incremental authorizations.
+On the authorization request the value of the incremental property has to be true and the status of the transaction has to be authorized.
```
Comment (jamie-barnett-cko) [inline source/payments/incremental-authorization.md.erb:7]: Do we want to add an example for this request?

File: source/refs/content/payments/_invoices.md.erb [hunk 2]
```diff
@@ -343,6 +343,48 @@ RFC3339 date at which the auto capture should be performed.
...
+$transaction = $invoice->incrementAuthorization("10.00");
+```
+
+```go
+<%= partial "header-go" %>
+tr, _ := iv.IncrementAuthorization("10.00")
+```
+
+It is possible to increment the authorization amount of a payment by submitting incremental authorizations.
+On the authorization request the value of the incremental property has to be true and the status of the transaction has to be authorized.
```
Comment (jamie-barnett-cko) [inline source/refs/content/payments/_invoices.md.erb:386]: Same as above do we want an example for this to help demonstrate the flow?

**Review by jamie-barnett-cko — COMMENTED** [review]
just a couple of comments but overall looks good!

File: source/payments/incremental-authorization.md.erb [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: Incremental authorization
+---
+
+# Incrementing an existing authorized invoice
+It is possible to increment the authorization amount of a payment by submitting incremental authorizations.
```
[the remark this answers is not in the document]
Comment (andrea-vasapollo-cko) [reply source/payments/incremental-authorization.md.erb]: thanks I changed the text

File: source/refs/content/payments/_invoices.md.erb [hunk 4]
```diff
@@ -343,6 +343,48 @@ RFC3339 date at which the auto capture should be performed.
...
+<%= partial "header-php" %>
+$transaction = $invoice->incrementAuthorization("10.00");
+```
+
+```go
+<%= partial "header-go" %>
+tr, _ := iv.IncrementAuthorization("10.00")
+```
+
+It is possible to increment the authorization amount of a payment by submitting incremental authorizations.
```
[the remark this answers is not in the document]
Comment (andrea-vasapollo-cko) [reply source/refs/content/payments/_invoices.md.erb]: thanks I changed the text

File: source/refs/content/payments/_invoices.md.erb (hunk 2 above)
Comment (andrea-vasapollo-cko) [reply source/refs/content/payments/_invoices.md.erb:386]: I am not sure here

File: source/payments/incremental-authorization.md.erb (hunk 1 above)
Comment (andrea-vasapollo-cko) [reply source/payments/incremental-authorization.md.erb:7]: maybe here where we describe the feature what do you think guys?

Comment (jamie-barnett-cko) [reply source/payments/incremental-authorization.md.erb:7]: yeah makes sense to me I wasn't particularly aware of the scope of the invoices page but I think you're right and we should only add the example here.

Comment (andrea-vasapollo-cko) [reply source/payments/incremental-authorization.md.erb:7]: ok mate I added the example to setup the authorize request with incremental

**Review by jamie-barnett-cko — APPROVED** [review]
LGTM

**Pull request merged**